idea/src/org/jetbrains/kotlin/idea/intentions/ConvertPropertyGetterToInitializerIntention.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.idea.intentions
+
+import com.intellij.openapi.editor.Editor
+import org.jetbrains.kotlin.psi.*
+import org.jetbrains.kotlin.psi.psiUtil.endOffset
+
+class ConvertPropertyGetterToInitializerIntention : SelfTargetingIntention<KtPropertyAccessor>(
+    KtPropertyAccessor::class.java, "Convert property getter to initializer"
+) {
+
+    override fun isApplicableTo(element: KtPropertyAccessor, caretOffset: Int): Boolean {
+        return element.isGetter && (element.parent as? KtProperty)?.hasInitializer() != true && element.singleExpression() != null
```

Comment:
There are properties which can have getter but cannot have initializer, e.g. interface properties. Please add extra check and tests for these cases.